library/Terminal42/ChangeLanguage/EventListener/DataContainer/PageFieldsListener.php

```diff
@@ -82,8 +82,9 @@ public function onSaveLanguageMain($value, DataContainer $dc)
                 [$value, $dc->id]
             );
 
-            if ($duplicates > 0) {
-                throw new \RuntimeException($GLOBALS['TL_LANG']['MSC']['duplicateMainLanguage']);
+            if (count($duplicates) > 0) {
+                $titles = $duplicates->fetchEach('title');
```

Comment:
I think the page id should be shown too, in case you have duplicate titles. Wdyt?

Reply:
Yeah, I though of that as well. That might be the best compromise 👍